src/app/(layout-with-navigation)/(simulation)/plan-du-site/page.tsx

```diff
@@ -14,26 +14,74 @@ export async function generateMetadata() {
 
 const links = {
   'Nos outils': {
-    'publicodes.planDuSite.bilan': '/simulateur/bilan',
-    'publicodes.planDuSite.profil': '/profil',
-    'publicodes.planDuSite.personas': '/personas',
-    'publicodes.planDuSite.actions': '/actions',
-    'publicodes.planDuSite.actionsPlus': '/actions/plus',
+    bilan: {
+      title: 'Le test',
+      href: '/simulateur/bilan',
+    },
```

Comment:
```suggestion
    'Le test': '/simulateur/bilan',
```

Reply:
Cependant, je pense que c'est utile de garder les clés i18n car sans elles on ne peut avoir de traductions...

J'avais essayé de résoudre le problème, mais je pense que la factorisation avec `LinkList` ne permet pas de bien utiliser la traduction.

En effet, pour qu'un texte à traduire soit parsé il faut qu'il soit hardcodé dans un appel à `t` ou bien dans un composant `<Trans>`... Je me demande si il ne faudrait pas to _défactoriser_ ici et se passer de `links` :thinking:

Reply:
ça fait mal de faire ça.. je teste un truc

Reply:
Okay ça fonctionne !